flytekit/image_spec/image_spec.py

```diff
@@ -153,9 +158,9 @@ def exist(self) -> bool:
                     f"    pip install --upgrade docker"
                 )
 
-            click.secho(f"Failed to check if the image exists with error : {e}", fg="red")
-            click.secho("Flytekit assumes that the image already exists.", fg="blue")
-            return True
+            click.secho(f"Failed to check if the image exists with error:\n {e}", fg="red")
+            click.secho(f"Flytekit assumes the image {self.image_name()} already exists.", fg="blue")
```

Comment:
I think this print statement belongs in `should_build` instead.

Reply:
That makes sense, I updated it, thanks!